src/command_modules/azure-cli-acr/README.rst

```diff
@@ -0,0 +1,205 @@
+                            json.
+        --query             : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        Get a container registry and show result in a table
+            az acr show -n <registry-name> -o table
+
+Update a container registry
+-------------
+::
+
+    Command
+        az acr update: Update a container registry.
+
+    Arguments
+        --name -n [Required]: The primary resource name.
+        --tags              : Multiple semicolon separated tags in 'key[=value]' format.  Use "" to
+                            clear existing tags.
+
+    Global Arguments
+        --debug             : Increase logging verbosity to show all debug logs.
+        --help -h           : Show this help message and exit.
+        --output -o         : Output format.  Allowed values: json, jsonc, list, table, tsv.  Default:
+                            json.
+        --query             : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        Update tags of a container registry and show result in a table
+            az acr update -n <registry-name> --tags key1=value1;key2=value2 -o table
+
+The catalog of repositories in the specified registry
+-------------
+::
+
+    Command
+        az acr catalog: The catalog of repositories in the specified registry.
+
+    Arguments
+        --name -n [Required]: The primary resource name.
+        --password          : The password used to log into the container registry.
+        --username          : The username used to log into the container registry.
+
+    Global Arguments
+        --debug             : Increase logging verbosity to show all debug logs.
+        --help -h           : Show this help message and exit.
+        --output -o         : Output format.  Allowed values: json, jsonc, list, table, tsv.  Default:
+                            json.
+        --query             : JMESPath query string. See http://jmespath.org/ for more information and
+                            examples.
+        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+
+    Examples
+        The catalog of repositories in a registry under the current subscription
+            az acr catalog -n <registry-name>
+        The catalog of repositories in any registry with credentials
+            az acr catalog -n <registry-name> --username <username> --password <password>
+
+The list of tags for a given repository in the specified registry
+-------------
+::
+
+    Command
+        az acr tags: The list of tags for a given repository in the specified registry.
```

Comment:
>tags: The list of tags for a given repository in the specified registry. [](start = 15, length = 72)

Why is this a separate command? Does the data not come down from the catalog command? Also, this should follow the 'noun verb' or 'verb-noun' pattern for command naming. In this case I would recommend 'show-tags' or 'list-tags' since there is only one thing you can do with tags.